app/_src/gateway/kong-enterprise/db-encryption.md

```diff
@@ -42,6 +42,8 @@ export KONG_KEYRING_STRATEGY=cluster
 export KONG_KEYRING_RECOVERY_PUBLIC_KEY=/path/to/generated/cert.pem
 ```
 
+If you are using a traditional Kong deployment, you need to add this configuration to both the control planes and data planes. If you are using a hybrid deployment, you only need this configuration on the control planes, since the data planes don't connect to the database.
```

Comment:
```suggestion
If you are using a hybrid deployment, you only need this configuration on the control planes, since the data planes don't connect to the database.
```
I'd completely remove the traditional mode but. In traditional mode, each node acts as both a control plane and data plane (so those terms don't even exist in trad mode) - there is no separation of duties like there is in hybrid.